### PR TITLE
TOS-95-Categories

### PR DESCRIPTION
**Pull request by darktend**
## Ticket
Link to ticket: https://teamchallengeonlinestore.atlassian.net/jira/software/projects/TOS/boards/1?assignee=609eb5f09b362f0069a70820&selectedIssue=TOS-95

## Description
*Please provide a small summary of your changes and how they resolved the issues in the ticket:*
Created a category component added pictures and links:Garden plants and indoor plants

## Screenshots
image

(prefer animated gif)

## Checklist
- [x] Did you test it?
- [x] Did you add a screenshot?
- [x] Did you fix the merge conflict?
- [x] Did you fill the PR template?
- [x] Did you review this PR on your own?

#### Last step if the ABOVE STEPS are COMPLETED!
- [x] Did you share the link in a group chat?

### CODE REVIEW

**Comment by KarlAnna** [comment]
The cards look different from the design

**Comment by dev1B** [comment]
Could you provide screenshot for mobile?

**Comment by dev1B** [comment]
What kind of changes did you make?

**Comment by dev1B** [comment]
The screenshot contains a bug, could you fix it?

**Review by dev1B — CHANGES_REQUESTED** [review]
Requested changes

**Comment by dev1B** [comment]
image
Please, review the checklist

**Comment by darktend** [comment]
@dev1B @KarlAnna review please

File: app/components/CategoriesList/CategoriesList.module.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.head {
```
Comment (dev1B) [inline app/components/CategoriesList/CategoriesList.module.scss:1]: `categoriesList/container/wrapper` could be better

File: app/components/CategoriesList/CategoriesList.module.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+.image {
+    flex: 1;
+    padding: 0 20px 0 0;
+    &:nth-child(2n) {
+        padding: 0;
+    }
+}
+
+.link {
```
Comment (dev1B) [inline app/components/CategoriesList/CategoriesList.module.scss:20]: This is not a link

File: app/components/CategoriesList/CategoriesList.module.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    max-width: 480px;
+    border-radius: 0;
+    border-bottom-right-radius: 10px;
+    border-bottom-left-radius: 10px;
+    width: 100%;
+    color: white;
+    padding: 18px 0;
+    text-align: center;
+
+    a {
```
Comment (dev1B) [inline app/components/CategoriesList/CategoriesList.module.scss:32]: better to create class, please

File: app/components/CategoriesList/CategoriesList.module.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        color: white;
+    }
+}
+
+@media (max-width: 980px) {
```
Comment (dev1B) [inline app/components/CategoriesList/CategoriesList.module.scss:37]: we don't have desktop version

File: app/components/CategoriesList/CategoriesList.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import Image from 'next/legacy/image';
+import { categories } from '@/utils/data';
```
Comment (dev1B) [inline app/components/CategoriesList/CategoriesList.tsx:2]: In this component better to use props instead of import

**Review by dev1B — CHANGES_REQUESTED** [review]

File: app/components/CategoriesList/CategoriesList.module.scss [hunk 6]
```diff
@@ -0,0 +1,35 @@
+.categoryList {
```
Comment (dev1B) [inline app/components/CategoriesList/CategoriesList.module.scss:1]: CategoriesList

File: app/components/CategoriesList/CategoriesList.module.scss [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    }
+}
+
+.wrapper {
+    background-color: #095F4D;
+    margin-top: -7px;
+    max-width: 480px;
+    border-radius: 0;
+    border-bottom-right-radius: 10px;
+    border-bottom-left-radius: 10px;
```
Comment (dev1B) [inline app/components/CategoriesList/CategoriesList.module.scss:27]: /* top-left | top-right | bottom-right | bottom-left */
border-radius: 1px 2px 3px 4px;

**Review by dev1B — CHANGES_REQUESTED** [review]

**Pull request merged**